requirements.txt

```diff
@@ -1,2 +1,6 @@
 # FairScale should only depends on torch, not things higher level than torch.
+# Note1: setup.py automatically reads this file to setup install time dependencies.
+# Note2: we use >= in this file but == in requirements-dev.txt for determinism
+#        in testing.
 torch >= 1.8.0
+numpy >= 1.22.0
```

Comment:
Since torch depends on numpy, we don't need to add this here. https://github.com/pytorch/pytorch/blob/master/requirements.txt#L5

Reply:
I thought that too. But pip doesn’t do that. If this line is removed. The test would fail. You can try pip install fairscale without any packages already installed. Import would fail.

Reply:
How did the installation of other packages needed for pytorch work earlier? How come we are finding this now?

Reply:
Found the answer. In CI we are downloading the pytorch wheel which contains all the dependencies. I think either we should list all the dependencies in requirements.txt or just leave it at torch>=1.8. What do you think @min-xu-ai ?